analyzers/src/SonarAnalyzer.CSharp/Rules/NormalizeStringsToUppercase.cs

```diff
@@ -30,23 +29,19 @@ namespace SonarAnalyzer.Rules.CSharp
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
     public sealed class NormalizeStringsToUppercase : DoNotCallMethodsCSharpBase
```

Comment:
We could do a bit more cleanup in this class:
- target-type constructors where possible
- move all parameters from `ShouldReportOnMethodCall` on the same line
- move operators from `ShouldReportOnMethodCall` conditions at the beginning of the lines